DesignDoc/RMA.md

```diff
@@ -1,3 +1,205 @@
+
+`case_open_time:date time:"Unix epoch time when consumer starts to request the RMA"`
+`case_approved_time:date time:"Unix epoch time when publisher approve to request the RMA"`
+`case_rejected_time:date time:"Unix epoch time when publisher reject the RMA"`
+
+#### part_shippment_status 
+
+Replacement / defective part shippment status
+
+`service_type:enum value:"Type of service: leased wave/dark fiber" enumeration: [LEASED_WAVE, DARK_FIBER]`
+
+#### part_type
+
+Replacement or defective parts
+
+#### service_impact
+
+Type of impact caused on the service because of the planned work.
+
+`service_impact:enum value:"Type of service impact." enumerations: [COMPLETE_OUTAGE, POSSIBLE_OUTAGE, INTERMITTENT_OUTAGE]`
+
+### impact_type
+
+Type of impact for the maintenance. If the outage would be complete, intermittent or possible.
+
+`impact_type:enum value:"If this impact is primary or secondary for a planned maintenance." enumerations: ['COMPLETE_OUTAGE', 'POSSIBLE_OUTAGE', 'INTERMITTENT_OUTAGE']`
+
+### impact_satus
+
+Status of the work.
+
+`impact_satus:enum value:"Status of the impact." 
+enumerations: [SCHEDULED, IN_PROGRESS, COMPLETED, CANCELED]
+`
+
+### impacts_related
+
+If there are other impacts related to the impact in notification, those could be listed here.
+
+`impact_type:repeated int:"Impacts related to the impact in the notification."`
+
+### physical_packet_link_info
+
+Physical packet link affected because of the change, this is for leased wave notifications.
+
+#### physical_packet_link_id
+
+Circuit ID of a link. ID could be stored independently at consumer and publisher. It would be ideal to have these matching between the two, but not necessary.
+
+`physical_packet_link_id:string:"Circuit identifying router port endpoints"`
+
+#### affected_capacity
+
+Link capacity (bps) affected because of the work.
+
+`affected_capacity:integer:"Affected traffic in bps"`
+
+### physical_packet_links_info
+
+If multiple links are affected because of the change, this field collects them together.
+
+` physical_packet_link_info:list of physical_packet_link_info:""
+
+`
+
+### ots_info
+
+Information regarding the Optical Transport Section. This is populated for a fiber maintenance.
+
+#### ots_id
+
+Fiber strand identifier.
+
+`ots_id:string:"Identifier for Optical Transport Section."`
+
+### ots_affected
+
+`ots_affected:repeated ots_info:"OTS affected because of the maintenance window."`
+
+### oms_info
+
+OMS information.
+
+#### oms_id
+
+OMS Identifier. `oms_id:string:"Identifier for Optical Multiplexing Section"`
+
+#### oms_affected
+
+oms_affected:list of oms_info:"OMS details regarding the maintenance"
+
+### close_code
+
+Status of the maintenance work closure.
+
+` 
+close_code:enum value:"Close code of the maintenance work under the impact" 
+enumerations: ['SERVICE_RESTORED', 'SERVICE_RESTORED_WITH_ISSUES', 'SERVICE_NOT_RESTORED', 'SERVICE_NOT_AFFECTED']
+
+`
+
+
+## Notification structure
```

Comment:
Please update this notification structure with your proposed fields.

Reply:
Added the problem description, and remove the duplicate with Fiber maintenance flow.